conf.yaml

```diff
@@ -0,0 +1,25 @@
+server:
+  server_secret: "37FUqWlvJhRgwPMM1mlHOGyPNwkVna3b"
```

Comment:
Add a comment here saying that this should be changed and using the default secret is insecure